### ISSUE TITLE
Type alias with option outputs invalid TS

### ISSUE THREAD

**Issue by azdavis** [1Password/typeshare#21]
Given this Rust input:

```rs
#[typeshare]
pub type Foo = Option<u32>;
```

The TypeScript output is:

```ts
export type Foo? = number;
```

This is invalid syntax.

**Comment by InquisitivePenguin** [comment]
Thank you for opening this issue. I'll look into why this is happening.

**Comment by InquisitivePenguin** [comment]
Fixed in #52. @azdavis or anyone else feel free to open this issue if it happens again.
